pyresample/bucket/__init__.py

```diff
@@ -456,6 +469,14 @@ def get_fractions(self, data, categories=None, fill_value=np.nan):
         return results
 
 
+def get_invalid_mask(data, fill_value):
```

Comment:
Small nitpick, maybe do a `_` prefix to this function to show that it isn't a public interface.

Reply:
Yep, agreed.

Reply:
Ok, done.